SoftLayer/CLI/vlan/list.py

```diff
@@ -65,7 +65,7 @@ def cli(env, sortby, datacenter, number, name, limit):
             vlan.get('vlanNumber'),
             vlan.get('fullyQualifiedName'),
             vlan.get('name') or formatting.blank(),
-            vlan.get('networkSpace').capitalize(),
+            vlan.get('networkSpace').capitalize() or formatting.blank(),
```

Comment:
Apparently Vlans with a null `networkSpace` are Direct Link vlans, so lets label them as such.

```suggestion
            vlan.get('networkSpace', 'DIRECT LINK').capitalize() ,
```

Also, I think `vlan.get('networkSpace').capitalize() or formatting.blank()` will still error on an empty `networkSpace` property. Setting a default for `vlan.get` should fix that.